### PR TITLE
Update to new zanata cli

### PR DESCRIPTION
**Pull request by marusak**
* [x] po-refresh

### CODE REVIEW

**Comment by cockpituous** [comment]
po-refresh in progress on 1-cockpit-11.
Log: https://209.132.184.41:8493/logs/po-refresh-234-20191031-101010/

**Comment by cockpituous** [comment]
po-refresh  done: https://github.com/cockpituous/cockpit-podman/commits/po-refresh-20191031-101111

**Comment by marusak** [comment]
Ok, refresh worked. It needed one hack, since it was failing when manifest did not declare line number. I just put there 0 since the manifest is not that big it would make any practical difference. Also I have no idea why this was not crashing in cockpit, where manifest also does not have any line number.

File: po/de.po [hunk 1]
```diff
@@ -24,7 +24,7 @@ msgstr "Autor"
 
 #: src/app.jsx:401
 msgid "Automatically start podman on boot"
-msgstr "podman automatisch beim Hochfahren starten"
+msgstr "Podman automatisch beim Hochfahren starten"
```
Comment (martinpitt) [inline po/de.po]: Hah, it picked up the capitalization question we were discussing before.

File: po/manifest2po [hunk 2]
```diff
@@ -92,7 +92,7 @@ function process_menu(menu) {
         if (menu[m].label) {
             push({
                 msgid: menu[m].label,
-                locations: [ filename ]
+                locations: [ filename + ":0" ]
```
Comment (martinpitt) [inline po/manifest2po]: This should get a comment, as it's not obvious what that :0 does.

Comment (marusak) [reply po/manifest2po]: Done

File: po/manifest2po [hunk 3]
```diff
@@ -92,7 +92,7 @@ function process_menu(menu) {
         if (menu[m].label) {
             push({
                 msgid: menu[m].label,
-                locations: [ filename ]
+                locations: [ filename + ":0" ] // Fake line number to zero
```
Comment (martinpitt) [inline po/manifest2po]: Well, that's the "what", which is quite obvious -- I meant document the "why", like "zanata crashes with BlahException when there is no line number"

File: po/manifest2po [hunk 4]
```diff
@@ -92,7 +92,8 @@ function process_menu(menu) {
         if (menu[m].label) {
             push({
                 msgid: menu[m].label,
-                locations: [ filename ]
+                locations: [ filename + ":0" ] // Fake line number because when no line number
+                // present `zanata pull` fails with `ValueError: not enough values to unpack`
```
Comment (marusak) [inline po/manifest2po]: ...and that's best I can do :D

**Pull request merged**